.gitignore

```diff
@@ -32,4 +32,6 @@ instance/
 
 #TODO add .env to your gitignore
 .env
-/server/.env
+./server/.env
+./client/.env
+server/utils/auth/client_secrets.json
```

Comment:
@bugatti12345 I will send the client_secret in slack, just put in under the same path